### PR TITLE
fix: quaratine and unquarantine column order

### PR DESCRIPTION
**Pull request by bao-ho**
## Description

Quarantine and unquarantine column order of sn-table so that the property can be stored and recovered correctly.

### CODE REVIEW

File: src/ext/conversion/export-properties.ts [hunk 1]
```diff
@@ -44,6 +44,8 @@ const exportProperties = (propertyTree: PropTree, hyperCubePath?: string): Expor
   setValue(expFormat, 'properties.qHyperCubeDef', qHyperCubeDef);
 
   conversion.quarantineProperty(expFormat.properties, 'qHyperCubeDef.columnWidths', 'straightTableColumnWidths');
+  conversion.quarantineProperty(expFormat.properties, 'qHyperCubeDef.qColumnOrder', 'straightTableColumnOrder');
```
Comment (LiKang6688) [inline src/ext/conversion/export-properties.ts:47]: qHyperCubeDef.qColumnOrder and qHyperCubeDef.columnOrder are same?

Comment (bao-ho) [reply src/ext/conversion/export-properties.ts:47]: yes, I assume columnOrder is an old property that has been replaced by qColumnOrder. We still need to support columnOrder for the old object to work, I think.

Comment (LiKang6688) [reply src/ext/conversion/export-properties.ts:47]: you are right, sn-table->old-table->sn-table

**Review by LiKang6688 — APPROVED** [review]
LGTM

File: src/ext/conversion/__test__/export-properties.spec.ts [hunk 2]
```diff
@@ -83,6 +83,8 @@ describe('exportProperties', () => {
         qHyperCubeDef: {
           qDimensions,
           qMeasures,
+          qColumnOrder: [0, 2, 1],
+          columnOrder: [0, 1],
```
Comment (LiKang6688) [inline src/ext/conversion/__test__/export-properties.spec.ts:87]: we make qColumnOrder and columnOrder same here to migrate confusion?

Comment (bao-ho) [reply src/ext/conversion/__test__/export-properties.spec.ts:87]: sure! I will fix that.

**Review by haxxmaxx — APPROVED** [review]
Nice that you added the tests :)

**Pull request merged**